setup.py

```diff
@@ -13,7 +13,7 @@
     author_email="pip@pykern.org",
     install_requires=[
         "argh>=0.26",
-        "black>=22",
+        "black==22.12",
```

Comment:
@robnagler black's updates are not backwards compatible for our purposes. Run on sirepo, the difference between 22.12 and the latest 23.1a1 is:

```
diff --git a/sirepo/template/flash.py b/sirepo/template/flash.py
index 048b361df..73bcd0286 100644
--- a/sirepo/template/flash.py
+++ b/sirepo/template/flash.py
@@ -418,7 +418,10 @@ def stateless_compute_update_lib_file(data):
             _SIM_DATA.flash_app_lib_basename(data.args.archiveLibId),
         ).copy(p)
     else:
-        simulation_db.simulation_dir(SIM_TYPE, data.args.simulationId,).join(
+        simulation_db.simulation_dir(
+            SIM_TYPE,
+            data.args.simulationId,
+        ).join(
             _SIM_DATA.flash_app_archive_basename()
         ).rename(p)
     return PKDict(

```

Reply:
So the fixed version will not force a rewrite of the repo?

Reply:
[Just these changes.](https://github.com/radiasoft/sirepo/pull/5295) mostly removing blank lines in function and class definitions

Reply:
What I was confused about was which changes were caused by which version. 23.1a1 will break more changes, but we can plan when that happens to avoid broken PR checks. For now let's fix the version to `black~=22.0`.

Reply:
c3b091879b8a8cd266d4d1fe691be99e1d88166e